go.mod

```diff
@@ -9,6 +9,53 @@ require (
 )
 
 require (
+	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
```

Comment:
I am surprised that we have to specify all these indirect dependencies. Why are these needed?

Reply:
This was just the result of doing `go get github.com/sigstore/rekor`. Do you have another way to get the sigstore library working?

Reply:
I think I only needed to run `go mod tidy` to get the required dependencies added to the `go.mod`.